openmetadata-ui/src/main/resources/ui/src/components/AddWebhook/AddWebhook.tsx

```diff
@@ -166,19 +180,31 @@ const AddWebhook: FunctionComponent<AddWebhookProps> = ({
     return retVal;
   };
 
+  const getHiddenEntitiesList = (entities: Array<string> = []) => {
+    if (entities.includes('*')) {
```

Comment:
`const allEntities="*"`
Store this value in constant and use that constant here and in other places.